durango/src/error.rs

```diff
@@ -0,0 +1,96 @@
+//! The Durango error
```

Comment:
Also it looks like this file got missed in the rebase of renaming Durange -> VeracruzClient